### PR TITLE
User can view their entries - Mohamed 

### PR DESCRIPTION
**Pull request by Mohamed-Elmessiry**

### CODE REVIEW

**Comment by Mohamed-Elmessiry** [comment]
Screen Shot 2021-04-01 at 1 28 07 PM
Screen Shot 2021-04-01 at 1 28 00 PM

**Comment by Mohamed-Elmessiry** [comment]
Screen Shot 2021-04-01 at 3 14 07 PM

**Comment by Mohamed-Elmessiry** [comment]
Screen Shot 2021-04-01 at 3 14 03 PM

File: css/styles.css [hunk 1]
```diff
@@ -121,3 +120,98 @@ button {
...
+  padding-top: 20px;
+}
+
+.new {
+  text-align: center;
+}
+
+.photo-label {
+  padding-top: 30px;
+}
```
Comment (codingcodymiller) [inline css/styles.css:185]: You should never have normal styling below a media query, please move this above your media query

File: css/styles.css [hunk 2]
```diff
@@ -121,3 +120,98 @@ button {
     width: 100%;
   }
 }
+
+.hide {
+  display: none;
+}
```
Comment (codingcodymiller) [inline css/styles.css]: There is already a `hidden` class rule in `layout.css`, you should remove this, and just use that `hidden` class

File: index.html [hunk 3]
```diff
@@ -46,6 +53,19 @@ <h2>New Entry</h2>
         </div>
       </form>
     </div>
+    <div class="viewTwo" data-view="entries">
```
Comment (codingcodymiller) [inline index.html]: Why `viewTwo`? How about `entries-list`?

File: index.html [hunk 4]
```diff
@@ -46,6 +53,19 @@ <h2>New Entry</h2>
...
+      <div class="entries">
+        <h2 class="hEntries">Entries</h2>
+        <div class="new-button">
+          <button class="new">NEW</button>
+        </div>
+
+      </div>
+      <ul class="dummy">
```
Comment (codingcodymiller) [inline index.html:64]: Better class name, please

File: js/main.js [hunk 5]
```diff
@@ -21,3 +21,78 @@ formSelector.addEventListener('submit', function (event) {
...
+  heading.textContent = entry.title;
+  listTwo.appendChild(heading);
+  var listThree = document.createElement('li');
+  listThree.textContent = entry.textbox;
+  divText.appendChild(listThree);
+
+  return divOne;
+}
+
+var dummySelector = document.querySelector('.dummy');
```
Comment (codingcodymiller) [inline js/main.js:54]: Better variable name, please

File: js/main.js [hunk 6]
```diff
@@ -21,3 +21,78 @@ formSelector.addEventListener('submit', function (event) {
...
+  displayList.className = 'dummy';
+  data.view = 'entries';
+  htwoEntries.className = 'hEntries';
+  newButton.className = 'new';
+
+  restore();
+
+});
+
+function restore(event) {
```
Comment (codingcodymiller) [inline js/main.js]: The name of this function doesn't describe its purpose. You should name this `changeView`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request closed without merging**